pagopa/io-web-profile
# [IOPID-1218] : Handle Expired Token Scenario

## Short description

The objective of this PR is to handle the scenario where an inactive user in session has an expired token. Upon refreshing the page with an expired token and receiving a 403 error, the user should be redirected to the login page, clearing the storage.

## List of changes proposed in this pull request

Clean the storage and redirect to login page in case of 403 handled by extractResponse  method

## How to test

Start the project and perform a login,

change the API called on Mockoon to return a 403 as a response from the profile page (or anywhere else), ensure that the endpoint where the response was modified is called.

Verify that the storage is cleared and that a redirect to the login page is performed
